### DISCUSSION TITLE
Self hosting uploading to wrong port.

### DISCUSSION THREAD

**Discussion by elvisdragonmao** [ente-io/ente#3866] [Q&A] [answered]
I'm trying to do self hosting on my raspberry pi. The login and the site are fine but when I tried to upload file it keeps trying to send to localhost:3200 instead of 192.168.1.92:3200.

I've tried:
```bash
NEXT_PUBLIC_ENTE_ENDPOINT=http://192.168.1.92:8080 \
    NEXT_PUBLIC_ENTE_ALBUMS_ENDPOINT=http://192.168.1.92:3002 \
    yarn build:photos
```

```bash
NEXT_PUBLIC_ENTE_ENDPOINT=http://192.168.1.92:8080 \
    NEXT_PUBLIC_ENTE_ALBUMS_ENDPOINT=http://192.168.1.92:3002 \
    yarn dev:photos
```

and also

```bash
em@raspberrypi:~/ente/server $ cat museum.yaml
internal:
    admins:
        - 1580559962386438
apps:
    public-albums: http://192.168.1.92:3002
```

But none of them work. Hope if anyone know what's the problem.

![圖片](https://github.com/user-attachments/assets/8a087337-67f7-468d-b2c6-9f305e0c2b6a)

**Comment by mngshm** [comment, marked as answer]
Hey, not sure if I am right, but public-albums is not the right place to configure this. That section in the `apps:` section in the yaml file is likely used to specify the base endpoints.

**Comment by elvisdragonmao** [comment]
Hi @mngshm . You're right. Been stuck for hours, it should be config at [server/configurations/local.yaml](https://github.com/ente-io/ente/blob/main/server/configurations/local.yaml). Finally success uploading.
```ymal
s3:
    # Override the primary and secondary hot storage. The commented out values
    # are the defaults.
    #
    #hot_storage:
    #    primary: b2-eu-cen
    #    secondary: wasabi-eu-central-2-v3
    b2-eu-cen:
        key:
        secret:
        endpoint: http://192.168.1.92:3200
        region:
        bucket:
```

**Reply by mngshm** [reply]
I kind of wanted to actually mark this as an answer. But I do not see the option. 

good that you found the fix.
